.github/workflows/cd.yml

```diff
@@ -3,21 +3,13 @@ name: CD
 on:
   workflow_dispatch:
     inputs:
-      version:
-        description: "which version to bump(major, minor, patch, premajor, preminor, prepatch, prerelease)"
-        required: false
-        default: "prerelease"
       preid:
         description: "preid name(alpha, rc, stable)"
-        required: false
-        default: "alpha"
-      force:
-        description: "force release even if no update(set --force-publish if necessary)"
-        required: false
-        default: ""
-      conventional-commits:
-        description: "bump up release version by lerna conventional commits(yes or no)"
         required: true
+        default: "rc"
+      skip-rc-version:
+        description: "skip bump up rc version by lerna conventional commits(yes or no)"
```

Comment:
what's the scenario to skip bump rc version in this CD pipeline?

Reply:
offline discussed, may be we could rename it like `enable-auto-versioning`?

Reply:
just rename it as "skip-version-rc" and modify the description
actually, this config work for rc version, so I think it's better add rc on it